src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -1146,17 +1160,42 @@ private static Attribute NewCmdletBindingAttribute(AttributeAst ast)
             return result;
         }
 
-        private static Attribute NewParameterAttribute(AttributeAst ast)
+        private static Attribute NewExperimentalAttribute(AttributeAst ast)
         {
-            CheckNoPositionalArgs(ast);
+            int positionalArgCount = ast.PositionalArguments.Count;
+            if (positionalArgCount != 2)
+            {
+                throw InterpreterError.NewInterpreterException(null, typeof(MethodException), ast.Extent,
+                    "MethodCountCouldNotFindBest", ExtendedTypeSystem.MethodArgumentCountException, ".ctor",
+                    positionalArgCount);
+            }
 
-            var cvv = new ConstantValueVisitor { AttributeArgument = true };
+            var (name, action) = GetFeatureNameAndAction(ast);
```

Comment:
Any way to declare the expected types of `name` and `action` here?

Reply:
I don't find a syntax for that :(

Reply:
Maybe `(string name, Action action) = GetFeatureNameAndAction(ast);`?

Reply:
Hmm, I remember I tried this and it didn't work. I obviously had some typos in my first try.
Will change.